NuRadioMC/EvtGen/generator.py

```diff
@@ -288,14 +289,15 @@ def get_energy_from_flux(Emin, Emax, n_events, flux):
 
     Returns: array of energies
     """
-
+    if(rnd is None):
+        rnd = np.random.default_rng()
```

Comment:
Don't you need an else statement so that `nrd = np.random.default_rng(seed)` in the case where a seed is passed? Or did I just miss it?

Reply:
no, `rnd` is an argument of the function. The `rnd` object is only intialized once in the main generator function and then passed to this helper function. I added this if statement in case this function is called stand alone for whatever reason.

Reply:
Oh, duh. Okay, thanks.

Reply:
in line 801 the `rnd` object is initialized for a given seed